extractor/src/main/java/org/schabi/newpipe/extractor/playlist/PlaylistInfo.java

```diff
@@ -42,12 +42,6 @@ public enum PlaylistType {
          */
         MIX_MUSIC,
 
-        /**
-         * A mix made only of streams from (or related to) the same channel, for example YouTube
-         * channel mixes
-         */
-        MIX_CHANNEL,
-
```

Comment:
Are you really sure that channel mixes are gone? Even if they are gone from tests, maybe they aren't on the whole platform. This concept may be applicable to other services too.

This is a breaking change, this should precised in the PR description, so you're also required to add if you tested the changes in NewPipe and if you want to make a PR in the app repo to fix breaking changes, if applicable (from the PR template).

Reply:
Yeah it might make sense to keep the code, you never know if YouTube might readd them

Reply:
I just now noticed that this is inside ``org.schabi.newpipe.extractor.playlist`` and not in inside ``youtube``. Thought it was in ``youtube`` (because these mix type only exist for YT and nothing else).

I will reintroduce the enum but deprecate it.